src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,8 @@
+package duke;
+
+public class DukeException extends Exception {
```

Comment:
If your bot name is Donald then ideally your Exception should be DonaldException rather than DukeException, same goes for the main class name!